mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -356,7 +353,7 @@ function resultsToTables(results, target, uri) {
     tableid = "#" + tableid;
     $(target).html(html);
     $(target).attr('data', uri);
-    //retreive albumcovers
+    //retrieve albumcovers
     for (i = 0; i < coversList.length; i++) {
         getCover(coversList[i][0], target + '-cover-' + coversList[i][1], 'small');
```

Comment:
Just to be clear, I meant the optimisation of calling mopidy.getImages just once, but with all the covers to fetch. Currently we do individual calls for each cover.